src/main/java/TestBase.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+
+import java.util.concurrent.TimeUnit;
+
+public class TestBase {
+    public static final int DEFAULT_TIMEOUT = 10;
+    public WebDriver driver;
+
+    protected String baseUrl;
+
+    public TestBase(String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    public void waitForAllElements() {
```

Comment:
Do not understand, why do you need such waiter?
You set up page load time and implicit waiter for your WebDriver instance, so remove this waiter.